### PR TITLE
Adding more events

### PR DESCRIPTION
**Pull request by jespino**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -594,3 +674,5 @@ All the data (past and coming) are available publicly in JSON:
 ### October
 
 * 26-29: [KubeCon + CloudNativeCon NA 2026](https://events.linuxfoundation.org/kubecon-cloudnativecon-north-america-2026/) - Los Angeles (USA)
+
+(2025-04-29T00:00:00Z) * 29-1: [NDC Melbourne 2025](https://ndcmelbourne.com) - Melbourne (Australia) <a href="https://sessionize.com/ndc-melbourne-2025"><img alt="CFP NDC Melbourne 2025" src="https://img.shields.io/static/v1?label=CFP&message=until%2017-November-2024&color=green"></a>
```
Comment (scraly) [inline README.md:678]:
```suggestion
* 29-1/5: [NDC Melbourne 2025](https://ndcmelbourne.com) - Melbourne (Australia) <a href="https://sessionize.com/ndc-melbourne-2025"><img alt="CFP NDC Melbourne 2025" src="https://img.shields.io/static/v1?label=CFP&message=until%2017-November-2024&color=green"></a>
```

**Review by scraly — APPROVED** [review]

**Comment by scraly** [comment]
thanks! :)

**Comment by sizovs** [comment]
@jespino huge list! 🙉 where did you find them?

**Comment by jespino** [comment]
@sizovs it is a combination of searching "sessionize.com" on twitter, checking out papercall.io, and sometimes also checking dev.events. Sometimes is just because I know they are something in my area and I look explicitly for that.

**Pull request merged**